FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -182,7 +181,7 @@ - (instancetype)initWithAppName:(NSString *)appName
       RCNPersonalization *personalization =
           [[RCNPersonalization alloc] initWithAnalytics:analytics];
```

Comment:
fyi, these warnings were introduced by me and will be addressed when migrating Analytics interop to its own pod.